src/hotspot/cpu/riscv/riscv.ad

```diff
@@ -9294,25 +9160,6 @@ instruct cmpL_reg_imm0_branch(cmpOp cmp, iRegL op1, immL0 zero, label lbl)
   ins_short_branch(1);
 %}
 
-instruct cmpL_reg_imm0_loop(cmpOp cmp, iRegL op1, immL0 zero, label lbl)
```

Comment:
I think this instruction should keep too.

Reply:
The current count loop iterations is1000(setting by `LoopStripMiningIter` ), and if it is too large, it will cause the GC pause too long, so I don't think there is any point in implementing the counted loop for Long type. However, I still respect your opinion and keep `cmpL loop` in risv.ad.